bofire/data_models/surrogates/mlp.py

```diff
@@ -18,4 +23,44 @@ class MLPEnsemble(TrainableBotorchSurrogate):
     weight_decay: Annotated[float, Field(ge=0.0)] = 0.0
     subsample_fraction: Annotated[float, Field(gt=0.0)] = 1.0
     shuffle: bool = True
-    scaler: ScalerEnum = ScalerEnum.STANDARDIZE
+    scaler: ScalerEnum = ScalerEnum.NORMALIZE
+
+    @classmethod
+    def is_output_implemented(cls, my_type: Type[AnyOutput]) -> bool:
+        """Abstract method to check output type for surrogate models
+        Args:
+            my_type: continuous or categorical output
+        Returns:
+            bool: True if the output type is valid for the surrogate chosen, False otherwise
+        """
+        return isinstance(my_type, (CategoricalOutput, ContinuousOutput))
+
+
+class RegressionMLPEnsemble(MLPEnsemble):
+    type: Literal["RegressionMLPEnsemble"] = "RegressionMLPEnsemble"
+    final_activation: Literal["identity"] = "identity"
+
+    @classmethod
+    def is_output_implemented(cls, my_type: Type[AnyOutput]) -> bool:
+        """Abstract method to check output type for surrogate models
+        Args:
+            my_type: continuous or categorical output
+        Returns:
+            bool: True if the output type is valid for the surrogate chosen, False otherwise
+        """
+        return isinstance(my_type, ContinuousOutput)
+
+
+class ClassificationMLPEnsemble(MLPEnsemble):
+    type: Literal["ClassificationMLPEnsemble"] = "ClassificationMLPEnsemble"
+    final_activation: Literal["softmax"] = "softmax"
```

Comment:
`output_sclaer: Literal[ScalerEnum.Indetity] = ScalerEnum.Indetity`

I do not know if this works, but we have to forbid that a user assigns an output scaler, if it not works via the Literal, we have to add a field validator for this.